clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -447,10 +457,17 @@ void PointerVariableConstraint::insertQualType(uint32_t TypeIdx,
     QualMap[TypeIdx].insert(RestrictQualification);
 }
 
-bool PointerVariableConstraint::emitArraySize(std::ostringstream &Pss,
+
+/*
+ *   Take an array or nt_array variable, determines if it is
+ *   a constant array, and if so emits the apprioate syntax for a
+ *   stack-based array. This functions also updates various flags.
```

Comment:
What are the flags for? I don't know what `AllArrays` means, or `ArrayRun` -- these are not self-explanatory.

Reply:
Also, I'm not sure about this comment syntax. Use `//` comments (see what the LLVM standard says, and what we are doing elsewhere in the code).

Reply:
Got rid of one of the booleans, wasn't actually being used, unsure why it wasn't caught in a warning.
Other two need to be independant